src/support/ios.c

```diff
@@ -1175,6 +1175,138 @@ char *ios_readline(ios_t *s)
     return ios_take_buffer(&dest, &n);
 }
 
+int ios_peek_number_of_where_tokens(ios_t *s)
+{
+    int result = 0;
+    int where_length = 5; // length("where")
+    int lookahead = 0;
+
+    while(1) {
```

Comment:
PS thanks to Stefan Karpinski for [this comment](https://discourse.julialang.org/t/could-we-have-a-do-while-loop-in-julia/24795/14). Has been helpful to me in many different circumstances!

Reply:
```suggestion
    while (1) {
```
Glad it's working out! It's kind of a relief when you realize you don't have to figure out the while loop condition right at the start of the loop 🙂